### PR TITLE
Switch CI to GitHub Actions

### PR DESCRIPTION
**Pull request by makaimann**

### CODE REVIEW

**Comment by makaimann** [comment]
Marked `do not merge` because should turn off Travis hooks and delete the `.travis.yml` file before merging.

File: .github/workflows/ci.yml
Comment (yoni206) [inline .github/workflows/ci.yml:15]: no bitwuzla?

File: .travis.yml
Comment (yoni206) [inline .travis.yml:78]: maybe consider just removing the file? if we need it later we know where to find it

**Review by yoni206 — APPROVED** [review]
LGTM modulo 2 optional suggestions.

File: .github/workflows/ci.yml
Comment (makaimann) [reply .github/workflows/ci.yml:15]: It's not public yet so there's no good way for me to pull it.

File: .travis.yml
Comment (makaimann) [reply .travis.yml:78]: Yeah I'll definitely delete it before merging, thanks!

**Pull request merged**
